internal/cmdUtils/workerType_test.go

```diff
@@ -0,0 +1,174 @@
+// COPYRIGHT 2024 FERMI NATIONAL ACCELERATOR LABORATORY
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmdUtils
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetWorkerConfigStringSlice(t *testing.T) {
```

Comment:
Should you have invoke a check that worker type is a valid string like you have in the actual version, somewhere around here? I guess perhaps if you know exactly what the test content is perhaps it isn't needed. Just a thought.

Reply:
You're right - that exactly why I don't have the extra check in here.